pkg/reconciler/util.go

```diff
@@ -0,0 +1,45 @@
+package reconciler
```

Comment:
I would suggest putting this `util.go` file under https://github.com/knative/serving/tree/main/pkg/networking.

Reply:
done